### PR TITLE
docs: regular updates (mid Oct.)

### LINKED ISSUE #605: import: docs don't mention that a revision can be a tag/branch
https://dvc.org/doc/commands-reference/import

The docs don't mention that `--rev` can be a tag or branch.

Ref: https://discordapp.com/channels/485586884165107732/563406153334128681/618485021342433290

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Fix #605 (among many other changes).

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
I don't understand why the file changes in this PR include changed already merged in https://github.com/iterative/dvc.org/pull/709/files...

**Comment by jorgeorpinel** [comment]
p.s. .eslintrc.json and all the files in src can safely be ignored/markes as viewed during review to save time.

File: static/docs/command-reference/add.md [hunk 1]
```diff
@@ -164,41 +164,45 @@ $ file .dvc/cache/d8/acabbfd4ee51c95da5d7628c7ef74b
...
 ```
 
-Taking a directory under DVC control as simple as taking a single file:
+Taking a directory under DVC control as simple as with a single file:
 
 ```dvc
 $ dvc add pics
-
-Computing md5 for a large directory pics/train/cats. This is only done once.
+Computing md5 for a large number of files. This is only done once.
```
Comment (jorgeorpinel) [inline static/docs/command-reference/add.md]: This is the line I was referring to in https://github.com/iterative/dvc.org/pull/679#discussion_r336149908.

**Comment by jorgeorpinel** [comment]
The last commit will fix #605.

File: static/docs/command-reference/version.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ The detail of `Binary` depends on the way DVC was downloading and
     distributions, like Ubuntu
   - Red Hat package (`.rpm`) - file used to install packages in some Linux based
     distributions, such as Fedora, CentOS, etc.
-  - PKG file (`.pkg`) - file used to install apps on macOS
```
Comment (shcheklein) [inline static/docs/command-reference/version.md]: I think `macOS` is the right way actually

Comment (jorgeorpinel) [reply static/docs/command-reference/version.md]: I think either way is correct. macOS is more of a logo/brand design, MacOS is the correct way to spell a proper noun. My change was more for consistency's sake.

image

Comment (jorgeorpinel) [reply static/docs/command-reference/version.md]: but lmk if you prefer macOS I'll replace all.

Comment (shcheklein) [reply static/docs/command-reference/version.md]: no, I'm fine with either :)

**Pull request merged**